velox/docs/develop/testing/spark-query-runner.rst

```diff
@@ -0,0 +1,75 @@
+==================
+Spark Query Runner
+==================
+
+Introduction
+------------
+
+The Spark Query Runner is a tool designed to facilitate the testing of Velox.
+It achieves this by executing SQL queries on Apache Spark and comparing the results
+with those produced by Velox. It helps ensure the correctness of Velox's computing
+against Spark and provides a method for identifying potential issues in Velox's
+implementation. Currently Spark-3.5.1 is used in Velox.
```

Comment:
> Currently Spark-3.5.1 is used in Velox.

It is probably inaccurate to say that Velox uses Spark as it doesn't. Maybe, "Spark Query Runner is designed to run against Spark-3.5.1."